silver/tests/spec/test_invoice.py

```diff
@@ -195,7 +195,7 @@ def test_get_invoice(self):
                 "state": "issued",
                 "proforma": "http://testserver/proformas/%s/" % invoice.related_document.pk,
                 "invoice_entries": [],
-                "pdf_url": invoice.pdf.url,
+                "pdf_url": None,
```

Comment:
A test with the url of the silver pdf-view would be nice, if not one with a direct url to the pdf file too.